tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3927,6 +3956,15 @@ <h2>Create a distance feature</h2>
           };
         };
 
+        function convertThreshold(point, origKey, threshold){
+          return {
+            x: (point.step ==
+              origKey) ? point.step + ' - original value' : point.step,
+            scalar: point.scalar,
```

Comment:
If below statement is correct then you may not need a scalar field here.

Reply:
See comment below